### PR TITLE
add "target" property to fab-button

### PR DESCRIPTION
**Pull request by bitflower**
#### Short description of what this resolves:
It adds a feature: Setting a `href` on a fab-button renders an `<a>`tag instead of a button. The `<a>` supports the attribute `target` which wasn't supported yet. This PR adds this feature.

#### Changes proposed in this pull request:

- fab-button can now have a `target` attribute to be applied to the anchor in case of a provided `href` value
-
-

**Ionic Version**:

**Fixes**: #

### CODE REVIEW

File: core/src/components/fab-button/fab-button.tsx [hunk 1]
```diff
@@ -44,6 +44,13 @@ export class FabButton implements ComponentInterface {
    */
   @Prop() href?: string;
 
+  /**
+   * The target attribute specifies where to open the linked document.
```
Comment (brandyscarney) [inline core/src/components/fab-button/fab-button.tsx]:
```suggestion
   * Specifies where to display the linked URL.
```

File: core/src/components/fab-button/fab-button.tsx [hunk 2]
```diff
@@ -44,6 +44,13 @@ export class FabButton implements ComponentInterface {
...
+   * Only applies if a href is provided too.
```
Comment (brandyscarney) [inline core/src/components/fab-button/fab-button.tsx]:
```suggestion
   * Only applies when an `href` is provided.
```

File: core/src/components/fab-button/fab-button.tsx [hunk 3]
```diff
@@ -44,6 +44,13 @@ export class FabButton implements ComponentInterface {
...
+   * The default options are: `"_blank"`, `"_self"`, `"_parent"`, `"_top"`and `{framename}`.
```
Comment (brandyscarney) [inline core/src/components/fab-button/fab-button.tsx]:
```suggestion
   * Special keywords: `"_blank"`, `"_self"`, `"_parent"`, `"_top"`.
```

**Comment by bitflower** [comment]
Nice, this fixes my circumtantial german language ;-)

Had run `npm run build` in the root before >:( Worked inside `core`...

I could possible help with finishing https://github.com/ionic-team/ionic/pull/17883
Are you guys at it already?

**Comment by brandyscarney** [comment]
@bitflower Thanks! We haven't started working on the generic version of #17883. If it's something you're interested in we'd appreciate the help! I think it would be nice if we could pull in the available attributes to pass down if possible. I haven't looked into the potential issues with this approach such as typing issues or not being able to update them dynamically (if that's even needed).

**Comment by bitflower** [comment]
Sure, this week is packed but I could stick my head into this next week for 1-2 hours?

**Comment by brandyscarney** [comment]
@bitflower Absolutely! Let me know if I can help. 🙂

**Comment by bitflower** [comment]
Hey! It stills reports me "Changes requested". Do you want to wait with the merge until all other components are done? 

Have a nice week end!

**Comment by brandyscarney** [comment]
Thanks so much for your work on this! I'm going to close this one and add you as a co-author on this PR: #17883

**Comment by bitflower** [comment]
Hey Brandy, that's awesome! I'm officially an ionic developer now ;-) Thanks and CU in the future.

**Pull request closed without merging**